modules/tensor_mechanics/doc/content/source/kernels/ADWeakPlaneStress.md

```diff
@@ -0,0 +1 @@
+!template load file=stubs/moose_object.md.template name=ADWeakPlaneStress syntax=/Kernels/ADWeakPlaneStress
```

Comment:
We're going to need full .md files to document these.  It looks like for the other AD models, we just did a copy/paste of the existing documentation and changed names.